cli/src/main/resources/help/embed-host-controller.txt

```diff
@@ -10,50 +10,72 @@ DESCRIPTION
-                     writes should be handled and appear as part
-                     of the CLI's own output to System.out. The
-                     default value is "discard". If "echo" is
-                     chosen and the CLI is running an interactive
-                     session, the user should recognize that
-                     asynchronous output from the server may
-                     interfere with the normal appearance of
-                     the CLI prompt.
-
- --timeout         - Maximum time, in seconds, to wait for the
-                     host controller to reach a running (i.e.
-                     fully started) state once the embedded server
-                     is created. If unspecified, the command will
-                     block for as long as necessary, with no upper
-                     limit. If the value is less than 1, the command
-                     will return as soon as the embedded server has
-                     reached the point in its boot where it is
-                     manageable by the CLI.
+ --host-config     - Name of the host configuration file to use (default is
+                     "host.xml")
+
+ --std-out         - How to handle output to System.out from the embedded host
+                     controller. Legal values are "discard" and "echo". The
+                     "discard" value indicates that writes to System.out from
+                     the server should be discarded. The "echo" value means
+                     writes should be handled and appear as part of the CLI's
+                     own output to System.out. The default value is "discard".
+                     If "echo" is chosen and the CLI is running an interactive
+                     session, the user should recognize that asynchronous
+                     output from the server may interfere with the normal
+                     appearance of the CLI prompt.
+
+ --timeout         - Maximum time, in seconds, to wait for the host controller
+                     to reach a running (i.e. fully started) state once the
+                     embedded server is created. If unspecified, the command
+                     will block for as long as necessary, with no upper limit.
+                     If the value is less than 1, the command will return as
+                     soon as the embedded server has reached the point in its
+                     boot where it is manageable by the CLI.
+
+ --empty-host-config  - Start the embedded host controller with an empty host
+                        configuration (host.xml). The hostname will be
+                     automatically derived from the current hostname.
+                     Configuration will be persisted when configuration changes
+                     are made. [See --temp-host-controller-name below for
+                     details on how to start an embedded host controller with a
+                     temporary name.]
```

Comment:
This can go when --temp-host-controller-name does. (Just pointing it out as it would be easy to miss when we get rid of --temp-host-controller-name.)